### PR TITLE
DSS-2512 clean up local files

### PR DESCRIPTION
**Pull request by andrew-may-cts**
To avoid file buildup, added a local file delete

### CODE REVIEW

File: to_data_library/data/transfer.py [hunk 1]
```diff
@@ -385,8 +400,10 @@ def _get_keys_in_s3_bucket(self, aws_session, bucket_name, prefix_name, wildcard
         regex = re.compile(wildcard)
 
         pages = paginator.paginate(Bucket=bucket_name, Prefix=prefix_name)
-        print(f'pages: {pages}')
+        print(f'Pages: {pages}')
```
Comment (peterwj5) [inline to_data_library/data/transfer.py:403]: is this for testing or logging? if loggin then can you use logger?

File: to_data_library/data/transfer.py [hunk 2]
```diff
@@ -385,8 +400,10 @@ def _get_keys_in_s3_bucket(self, aws_session, bucket_name, prefix_name, wildcard
...
+
         for page in pages:
+            print(f'Page: {page}')
```
Comment (peterwj5) [inline to_data_library/data/transfer.py:406]: testing or logging??

File: to_data_library/data/transfer.py (hunk 1 above)
Comment (andrew-may-cts) [reply to_data_library/data/transfer.py:403]: testing - I'll remove

File: to_data_library/data/transfer.py (hunk 2 above)
Comment (andrew-may-cts) [reply to_data_library/data/transfer.py:406]: as above

**Review by peterwj5 — APPROVED** [review]
LGTM - great addition of more logging

**Pull request merged**